website/content/maintainers/community/contributor-growth-framework/open-source-roadmaps.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Roadmaps as a way to encourage contributions"
+date: 2023-06-28
+description: >
+  A guide to open-source project roadmaps.
+---
+
+# Importance of Roadmaps in Open Source Projects
+
+A roadmap is a strategic plan that outlines the goals and direction of an open source project. For open source projects, having a well-defined roadmap is crucial.
+
+- A roadmap provides a **clear vision and purpose for the project**, helping contributors understand the project's objectives and direction. This clarity motivates them to work towards common goals.
+- Roadmaps **aid project management** by setting priorities, allocating resources, and coordinating efforts. They ensure that development tasks are well-organized and yield meaningful outcomes.
+- With many feature requests and contributions, roadmaps **help prioritize tasks** based on their importance and alignment with project goals. This keeps development efforts focused on key areas.
+- Transparently sharing the roadmap **attracts contributors** by showcasing ongoing efforts and how their skills fit into the project's future.
+- Roadmaps foster trust by **offering transparency into development plans and progress**, ensuring contributors feel confident in the project's management and also good to guide the users and downstream adoptions. Users and companies can receive advanced notifications about upcoming developments, enabling them to strategically devise their individualized utilization strategies for the software.
+- Roadmaps **ensure development aligns with user demands**, enhancing the overall user experience.
```

Comment:
```suggestion
- Roadmaps may allow the project to **align its development activities with adopter, user, and contributor feedback**, enhancing the overall experience for everyone.
```
I'm suggesting this rewrite as I think it is more favorable for collaboration between users and the projects, rather than projects in service to users. I know I'm guilty of using this phrasing in the past.